src/OpenTelemetry/BatchExportProcessor.cs

```diff
@@ -38,7 +38,6 @@ public abstract class BatchExportProcessor<T> : BaseExportProcessor<T>
 
         private readonly CircularBuffer<T> circularBuffer;
         private readonly int scheduledDelayMilliseconds;
-        private readonly int exporterTimeoutMilliseconds;
```

Comment:
@alanwest @cijothomas @utpilla Is this indicative of a bug? Should we be using this value somewhere?

Reply:
missing [feature](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/sdk.md#span-processor:~:text=exportTimeoutMillis%20%2D%20how%20long%20the%20export%20can%20run%20before%20it%20is%20cancelled.%20The%20default%20value%20is%2030000.)?

Reply:
> missing [feature](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/sdk.md#span-processor:~:text=exportTimeoutMillis%20%2D%20how%20long%20the%20export%20can%20run%20before%20it%20is%20cancelled.%20The%20default%20value%20is%2030000.)?

Looks like it. We do let the user configure this value for the exporters.

Reply:
https://github.com/open-telemetry/opentelemetry-dotnet/issues/1735

Reply:
I revert this commit
Commit SHA (8380e31c1665e1458e03eb85905f11e44d8b7bac)